devportalservice/devportalservice.go

```diff
@@ -55,61 +92,44 @@ type cookie struct {
 	ForDomain *bool  `json:"for_domain,omitempty"`
 }
 
-// SessionData will fetch the session from Bitrise for the connected Apple developer account
-// If the BITRISE_PORTAL_DATA_JSON is provided (for debug purposes) it will use that instead.
-func SessionData() (string, error) {
-	p, err := getDeveloperPortalData(os.Getenv("BITRISE_BUILD_URL"), os.Getenv("BITRISE_BUILD_API_TOKEN"))
-	if err != nil {
-		return "", err
-	}
-
-	cookies, err := convertDesCookie(p.SessionCookies["https://idmsa.apple.com"])
-	if err != nil {
-		return "", err
-	}
-	return strings.Join(cookies, ""), nil
+// AppleDeveloperConnection ...
```

Comment:
NOTE: If used from other steps too, would be nice to replace ... with documentation.